pkg/kube/secretcontroller/secretcontroller_test.go

```diff
@@ -83,13 +84,13 @@ func mockCreateInterfaceFromClusterConfig(_ *clientcmdapi.Config) (kubernetes.In
 
 func verifyControllerDeleted(t *testing.T, timeoutName string) {
 	pkgtest.NewEventualOpts(10*time.Millisecond, 5*time.Second).Eventually(t, timeoutName, func() bool {
-		return testDeleteControllerCalled == true
+		return atomic.AddInt32(&testDeleteControllerCalled, 0) == 1
```

Comment:
use atomic.LoadInt32?

Reply:
read the docs wrong and missed that, thanks